R-packages/covidcast/R/plot.R

```diff
@@ -252,6 +253,80 @@ plot_choro = function(x, time_value = NULL, include = c(), range,
   geo = df$geo
   names(val) = geo
 
+  # Make background layer for MSA and HRR maps which are incomplete
+  if ((attributes(x)$metadata$geo_type == "msa") ||
```

Comment:
I've put repeated sections with shifting and scaling into helper functions.  I have not done it further because I will be replacing the state and county sections in my next PR, which will eliminate a lot of repetition because they will be doing the same thing.

Reply:
@sgsmob let me know if the changes work.